fluent/sender.py

```diff
@@ -201,7 +208,12 @@ def _reconnect(self):
                     sock.settimeout(self.timeout)
                     sock.connect(self.host[len('unix://'):])
                 else:
-                    sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                    if self._is_ipv4_host():
+                        sock = socket.socket(socket.AF_INET,
+                                             socket.SOCK_STREAM)
+                    else:
+                        sock = socket.socket(socket.AF_INET6,
+                                             socket.SOCK_STREAM)
                     sock.settimeout(self.timeout)
```

Comment:
Does it need to call  `_is_ipv4_host()` in every `_make_packet`?
It may be able to reuse checked status.